admin/src/scenes/plan-transport/ligne-bus/ListeDemandeModif.js

```diff
@@ -1,5 +1,333 @@
+                  </div>
+                )}
+              />
+            </div>
+          </div>
+        </ReactiveBase>
+      </div>
+    </>
+  );
 }
+
+const Line = ({ modification, tagsOptions }) => {
+  const history = useHistory();
+  const [open, setOpen] = React.useState(false);
+  const refContainer = React.useRef(null);
+  const refChildren = React.useRef(null);
+
+  function getlink(userId, role) {
+    if (Object.values(ROLES).includes(role)) return `/user/${userId}`;
+    if (role === "Volontaire") return `/volontaire/${userId}`;
+    return null;
+  }
+
+  React.useEffect(() => {
+    const handleClickOutside = (event) => {
+      if (refContainer.current && refContainer.current?.contains(event.target)) {
+        if (refChildren.current && !refChildren.current?.contains(event.target)) {
+          setOpen((open) => !open);
+        }
+      } else {
+        setOpen(false);
+      }
+    };
+    document.addEventListener("click", handleClickOutside, true);
+    return () => {
+      document.removeEventListener("click", handleClickOutside, true);
+    };
+  }, []);
+  return (
+    <>
+      <hr />
+      <div className="flex py-4 items-center px-4 hover:bg-gray-50 gap-6">
+        <div className="w-[35%] flex flex-col gap-1">
+          <div className="line-clamp-3 text-sm text-[#242526] text-start">{modification.requestMessage}</div>
+          <div className="text-sm text-[#738297]">{dayjs(modification.createdAt).locale("fr").format("DD/MM/YYYY • HH:mm")}</div>
+        </div>
+        <div className="w-[12%] flex items-center gap-2">
+          <div className="text-base text-[#242526] font-medium">{modification.lineName}</div>
+          <ExternalLink className="h-3 w-3 text-[#9CA3AF] cursor-pointer hover:scale-110" onClick={() => history.push(`/ligne-de-bus/${modification.lineId}`)} />
+        </div>
+        <div className="w-[23%] flex pr-4">
+          {modification?.tagIds?.length === 0 ? (
+            <div className=" text-sm text-gray-700 p-2 bg-gray-100 leading-4 rounded-lg">Non renseigné</div>
+          ) : (
+            <div className="flex items-center gap-1">
+              <div className="px-2 py-1 rounded-lg bg-[#E8EDFF]">
+                <div className="text-sm text-[#0063CB]  line-clamp-2">{tagsOptions.find((option) => option.value === modification.tagIds[0])?.label}</div>
+              </div>
+              {modification.tagIds.length > 1 && (
+                <div
+                  ref={refContainer}
+                  className="relative flex justify-center items-center text-sm text-[#0063CB] h-full leading-4 px-2 py-1 rounded-lg bg-[#E8EDFF] cursor-pointer">
+                  +{modification.tagIds.length - 1}
+                  <div ref={refChildren} className={`absolute top-[105%] left-[95%] bg-white shadow-lg rounded-xl z-10 w-96 ${!open ? "hidden" : ""}`}>
+                    <div className="flex flex-col p-3 gap-2">
+                      {modification.tagIds.map((value, index) => (
```

Comment:
UX: le bouton marque +1 par exemple et quand on ouvre la dropdwn on voit le +1 + le tag déjà affiché. Du coup, on en a 2 dans la dropdown. Je trouve ça hyper bizarre de cliquer sur +1 et dans voir 2 (perso, je ne remettrais pas la première dans la dropown) 
```suggestion
                      {modification.tagIds.slice(1).map((value, index) => (
```

Reply:
La maquette est comme ca

Reply:
![image](https://user-images.githubusercontent.com/50077368/207814183-83c4b024-141e-4853-bcc0-3de1e4df8700.png)

Reply:
ah ouai... c'était juste une impression à l'utilisation... Mais pas de soucis.